handlers/discord/discord.go

```diff
@@ -0,0 +1,243 @@
+package discord
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+
+	"strings"
+
+	"github.com/nyaruka/courier"
+	"github.com/nyaruka/courier/handlers"
+	"github.com/nyaruka/courier/utils"
+	"github.com/nyaruka/gocommon/urns"
+	"github.com/pkg/errors"
+)
+
+const (
+	contentURLEncoded = "urlencoded"
+	contentJSON       = "json"
+
+	configMOFromField = "mo_from_field"
```

Comment:
Is this really a configuration item? Looking at your proxy seems these are probably always going to be static no? May as well just turn these into consts then instead of making them configurable.